ArduPlane/GCS_Plane.cpp

```diff
@@ -8,17 +8,8 @@ uint8_t GCS_Plane::sysid_this_mav() const
 
 void GCS_Plane::update_vehicle_sensor_status_flags(void)
 {
-    // airspeed
-    const AP_Airspeed *airspeed = AP_Airspeed::get_singleton();
-    if (airspeed && airspeed->enabled()) {
-        control_sensors_present |= MAV_SYS_STATUS_SENSOR_DIFFERENTIAL_PRESSURE;
-    }
-    if (airspeed && airspeed->enabled() && airspeed->use()) {
-        control_sensors_enabled |= MAV_SYS_STATUS_SENSOR_DIFFERENTIAL_PRESSURE;
-    }
-    if (airspeed && airspeed->all_healthy()) {
-        control_sensors_health |= MAV_SYS_STATUS_SENSOR_DIFFERENTIAL_PRESSURE;
-    }
+    // Update common flags
+    GCS::update_vehicle_sensor_status_flags();
```

Comment:
Don't call this; insert the airspeed code here: https://github.com/ardupilot/ardupilot/blob/master/libraries/GCS_MAVLink/GCS.cpp#L258

Reply:
Thanks. Fixed.